langgraph/src/channels/base.ts

```diff
@@ -18,9 +18,8 @@ export abstract class BaseChannel<
    * @param {CheckpointType | undefined} initialValue
    * @returns {this}
    */
-  abstract empty(
-    checkpoint?: CheckpointType,
-    initialValueFactory?: () => CheckpointType
```

Comment:
`initialValueFactory` is removed from `BaseChannel`, but it remains in `BinaryOperatorAggregate`.